packages/wallet-backend/src/asset/asset.route.ts

```diff
@@ -1,9 +1,15 @@
 import express from 'express'
 import passport from 'passport'
-import { getById as getAssetById, list as listAsset } from './asset.service'
+import {
+  create,
+  getById as getAssetById,
+  list as listAsset
+} from './asset.service'
 
 export const assetRouter = express.Router()
 
+//* Testing purpose: In order to properly create assets from postman
+assetRouter.post('', create)
```

Comment:
We don't need this anymore. We can now access Rafiki's Admin UI to properly create assets. (http://localhost:3012/)